### PR TITLE
Adds the Barrel fire, tweaks microwave code

### PR DESCRIPTION
**Pull request by zombiemimic**
## About The Pull Request
Adds the Barrel fire. Come, rest your feet, and cook those roachburgers.
campfire

## Why It's Good For The Game
Maint stove and maint vibes

## Changelog
:cl:
add: Vagabonds have put their brain cell (singular) together and discovered how to make fire, and how to put it into a barrel
del: removed the microwave breaking
/:cl:

### CODE REVIEW

**Comment by Sparrowbird611** [comment]
Looks nice. Time for the maintenance grill shop.

File: code/game/machinery/kitchen/microwave.dm [hunk 1]
```diff
@@ -406,3 +409,55 @@
 		if("dispose")
 			dispose()
 	return
+
+/obj/machinery/microwave/campfire
+	name = "Barrel Fire"
```
Comment (Firefox13) [inline code/game/machinery/kitchen/microwave.dm:414]:
```suggestion
	name = "burn barrel"
```

File: code/game/machinery/kitchen/microwave.dm [hunk 2]
```diff
@@ -406,3 +409,55 @@
...
+	idle_power_usage = 0
+	active_power_usage = 0
+	dinger = FALSE
+
+
+
+/obj/machinery/microwave/campfire/start()
+	..()
+	playsound(loc, 'sound/effects/flare.ogg', 50, 1)
+	visible_message(SPAN_NOTICE("The fire is stoked up."), SPAN_NOTICE("You hear a crackling fire."))
```
Comment (Firefox13) [inline code/game/machinery/kitchen/microwave.dm:427]: How those messsage look in-game?

File: code/game/machinery/kitchen/microwave.dm (hunk 1 above)
Comment (Firefox13) [reply code/game/machinery/kitchen/microwave.dm:414]: Also, shouldn't you add `desc` to it?

File: code/datums/craft/recipes/furniture.dm [hunk 3]
```diff
@@ -203,3 +203,11 @@
 /datum/craft_recipe/furniture/comfy_chair/green
 	name = "green comfy chair"
 	result = /obj/structure/bed/chair/comfy/green
+
+/datum/craft_recipe/furniture/campfire
+	name = "Barrel Fire"
```
Comment (Firefox13) [inline code/datums/craft/recipes/furniture.dm:208]:
```suggestion
	name = "burn barrel"
```

File: code/game/machinery/kitchen/microwave.dm [hunk 4]
```diff
@@ -406,3 +409,55 @@
...
+
+/obj/machinery/microwave/campfire/muck_finish()
+	..()
+	playsound(loc, 'sound/effects/flare.ogg', 50, 1)
+	icon_state = "barrelfire"
+	set_light(0)
+/obj/machinery/microwave/campfire/broke()
+	..()
+	icon_state = "barrelfire"
+	set_light(0)
```
Comment (Firefox13) [inline code/game/machinery/kitchen/microwave.dm:463]: newline

**Review by Firefox13 — CHANGES_REQUESTED** [review]
We could use different sounds rather than those that are used in flares, they sound pretty wack.

File: code/game/machinery/kitchen/microwave.dm [hunk 5]
```diff
@@ -406,3 +396,52 @@
 		if("dispose")
 			dispose()
 	return
+
+/obj/machinery/microwave/campfire
+	name = "Burn Barrel"
```
Comment (Firefox13) [inline code/game/machinery/kitchen/microwave.dm:401]:
```suggestion
	name = "burn barrel"
```
Please avoid using uppercases

File: code/datums/craft/recipes/furniture.dm [hunk 6]
```diff
@@ -203,3 +203,11 @@
 /datum/craft_recipe/furniture/comfy_chair/green
 	name = "green comfy chair"
 	result = /obj/structure/bed/chair/comfy/green
+
+/datum/craft_recipe/furniture/campfire
+	name = "Burn Barrel"
```
Comment (Firefox13) [inline code/datums/craft/recipes/furniture.dm:208]:
```suggestion
	name = "burn barrel"
```
ditto, uppercase

**Pull request merged**